webpack/assets/javascripts/react_app/routes/RoutingService.js

```diff
@@ -0,0 +1,42 @@
+import { visit } from '../../foreman_navigation';
+
+let currentPath = window.location.href;
+
+/**
+ * a Helper function for rendering a route
+ * @param {Function} renderFn - a component's rendering function
+ * @param {Object} props - routing props
+ * @param {Function} beforeRenderingCallback - a callback to run before the rendering, if it returns false the rendering will terminate
+ */
+export const renderRoute = (
+  renderFn,
+  props,
+  beforeRenderingCallback = () => true
+) => {
```

Comment:
can you pass the args in an object? it's a better practice for future changes.

Reply:
this is a helper function with jsdoc documentation for the arguments, this is how all our other helper functions look like, I don't see a benefit for object here.